Utils/DynamicMethodDefinition.CopyMethodToDefinition.cs

```diff
@@ -1,305 +1,305 @@
+                    // this can still throw if the assembly is a dynamic one, but if that's broken, you have bigger issues
+                    string filePath = moduleFrom.Assembly.Location;
+                    if (!File.Exists(filePath)) {
+                        // in this case, the fallback cannot be followed, and so throwing the original error gives the user information
+                        throw;
+                    }
+
+                    // TODO: make this cached somehow so its not read and re-opened a bunch
+                    using (AssemblyDefinition assembly = AssemblyDefinition.ReadAssembly(filePath, new ReaderParameters {
+                        ReadingMode = ReadingMode.Deferred
+                    })) {
+                        ModuleDefinition module = assembly.Modules.First(m => m.Name == moduleFrom.Name);
+                        // this should only fail if the token itself is somehow wrong
+                        MemberReference reference = (MemberReference) module.LookupToken(token);
+                        // the explicit casts here are to throw if they are incorrect
+                        // normally the references would need to be imported, but moduleTo isn't written to anywhere
+                        switch (resolveMode) {
+                            case TokenResolutionMode.Type:
+                                return (TypeReference) reference;
+
+                            case TokenResolutionMode.Method:
+                                return (MethodReference) reference;
+
+                            case TokenResolutionMode.Field:
+                                return (FieldReference) reference;
+
+                            case TokenResolutionMode.Any:
+                                return reference;
+
+                            default:
+                                throw new NotSupportedException($"Invalid TokenResolutionMode {resolveMode}");
+                        }
+                    }
+                }
+            }
+
+            Instruction GetInstruction(int offset) {
+                int last = bodyTo.Instructions.Count - 1;
+                if (offset < 0 || offset > bodyTo.Instructions[last].Offset)
+                    return null;
+
+                int min = 0;
+                int max = last;
+                while (min <= max) {
+                    int mid = min + ((max - min) / 2);
+                    Instruction instr = bodyTo.Instructions[mid];
+
+                    if (offset == instr.Offset)
+                        return instr;
+
+                    if (offset < instr.Offset)
+                        max = mid - 1;
+                    else
+                        min = mid + 1;
+                }
+
+                return null;
+            }
+
+        }
+
+        private enum TokenResolutionMode { 
+            Any,
+            Type,
+            Method,
+            Field
+        }
+
+    }
+}
```

Comment:
What was changed in `Utils/DynamicMethodDefinition.CopyMethodToDefinition.cs`? According to GitHub the entire file was replaced?

Reply:
I *think* it was line endings, though I'm not completely sure. VS says that that file has LF endings, so it might have CRLF on master.

Reply:
Would you be willing to rebase to fix this? It's not a big deal otherwise :)